aten/src/ATen/cudnn/Handles.cpp

```diff
@@ -19,7 +19,10 @@ struct Handle {
   }
   ~Handle() {
     if (handle) {
+#ifdef NO_CUDNN_DESTROY_HANDLE
```

Comment:
Woah, what's this? Why is this not working in fbcode?

Reply:
don't know, but this is existing behavior

Reply:
this is because of something dumb in the ordering of destruction. Sometimes `atexit`, the cuda context (or something) would already be destroyed by the time this gets destroyed. It happens in fbcode setting. @colesbury and I decided to not destroy the handle as a workaround.